pcp/Platform.c

```diff
@@ -699,6 +703,17 @@ bool Platform_getNetworkIO(NetworkIOData* data) {
    return true;
 }
 
+void Platform_getFileDescriptors(double* used, double* max) {
+   *used = NAN;
+   *max = 65536;
+
+   pmAtomValue value;
+   if (PCPMetric_values(PCP_VFS_FILES_COUNT, &value, 1, PM_TYPE_32) != NULL)
+      *used = value.l;
+   if (PCPMetric_values(PCP_NET_RECVB, &value, 1, PM_TYPE_32) != NULL)
+      *max = value.ull;
```

Comment:
Shouldn't they be read from the same field in `value`? They got the same `PM_TYPE_32` as their type after all …